UnityProject/Assets/Scripts/HealthV2/Living/BodyPartMutations.cs

```diff
@@ -210,10 +210,6 @@ private IEnumerator ProcessChangeToSpecies(PlayerHealthData NewSpecies, GameObje
 		Chat.AddExamineMsgFromServer(RelatedPart.OrNull()?.HealthMaster.OrNull()?.gameObject,
 			$" Your {RelatedPart.gameObject.ExpensiveName()} Morphs into a {SpawnedBodypart.gameObject.ExpensiveName()}");
 
-		foreach (var itemSlot in SpawnedBodypart.OrganStorage.GetItemSlots())
```

Comment:
What does it do afterwards?

Reply:
oh yeah, I've set it up that now you have to specify that subbody parts Spawn So therefore this is no longer needed